env/.env.production

```diff
@@ -0,0 +1 @@
+REACT_APP_FHIR_BASE_URL=https://localhost:9443/fhir-server/api/v4
```

Comment:
Is it ok to leave localhost here? (we might need to change that later)

Reply:
Yup, `.env.development` values are used when executing the developer build, e.g.  `npm run start` and `.env.production` values are used when executing the production build created by `npm run build`. We'll have to change this value once we have a hosted FHIR server.